taskvine/src/manager/vine_manager.c

```diff
@@ -1235,12 +1375,15 @@ static int fetch_output_from_worker(struct vine_manager *q, struct vine_worker_i
 		}
 	}
 
-	/* XXX: temp fix. Make this a tunable parameter and/or make it more sophisticated */
-	if (w->forsaken_tasks > VINE_DEFAULT_MAX_FORSAKEN_PER_WORKER && w->total_tasks_complete == 0) {
-		debug(D_VINE, "Disconnecting worker that keeps forsaking tasks %s (%s).", w->hostname, w->addrport);
-		handle_failure(q, w, t, VINE_WORKER_FAILURE);
-		return 0;
-	}
+	vine_task_info_add(q, t);
```

Comment:
Isn't this part of the `exit_alert` above?

Reply:
this one remains as `exit_alert` is not called.